src/app/add-book/add-book.component.ts

```diff
@@ -30,23 +30,18 @@ export class AddBookComponent implements OnInit {
     
   }
 submit(data:any){
-  this.serv.createELEMENT_DATA({...data.value,bookName:data.value['bookName']}).subscribe(s =>{
-   window.location.reload();
-    
-    
-  });
+  this.serv.createELEMENT_DATA({...data.value,bookName:data.value['bookName']}).subscribe()
+  window.location.reload();
```

Comment:
this is wrong, `window.location.reload(); ` should be inside subscribe only. I just said to remove the value declaration